index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Portfolio</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <img src="assets/logo.svg" id="logo" alt="Logo of page">
+        <nav>
+            <img src="assets/nav-burger.svg" class="nav-btn" alt="Button to navigate">
+        </nav>
+    </header>
+    <main class>
+        <section class="headline">
+            <h1 class="primary">
+                <span class="color1">
+                    Hey there. <br>
+                    I'm Fashina. <br>
+                </span>
+                <span class="color2">I'm a software developer</span>
+            </h1>
+            <p class="supporting-text">
+                I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hesitate to contact me.
+            </p>
+            <div class="icons">
+                 <a href="https://twitter.com/@KingFash14"><img src="assets/twitter.svg" alt="Twitter"></a> 
+                 <a href="https://www.linkedin.com/in/fashina-segun-031206233"><img src="assets/linkedin.svg" alt="LinkedIn"></a> 
+                 <a href="#"><img src="assets/medium.svg" alt="medium"></a> 
+                 <a href="https://github.com/Kingfash084412"><img src="Assets/github.svg" alt="GitHub"></a> 
+                 <a href="#"><img src="assets/angellist.svg" alt="Angellist"></a>
+            </div>
```

Comment:
- [ ] Please 🙏🏽 use the `ul` and `li` tags for the best practices of HTML & CSS, the social icons is an unordered list.